core/templates/components/summary-tile/collection-summary-tile.component.spec.ts

```diff
@@ -217,7 +217,7 @@ describe('Collection Summary Tile Component', () => {
     const dateTimeSpy = spyOn(dateTimeFormatService, 'getRelativeTimeFromNow')
       .and.returnValue('a few seconds ago');
 
-    component.getLastUpdatedMsec = new Date().getUTCMilliseconds();
+    component.getLastUpdatedMsec = Date.now();
```

Comment:
Add a comment specifying the format of the date that will be returned

Reply:
done